imports/plugins/included/search-mongo/server/methods/transformations.js

```diff
@@ -1,5 +1,10 @@
+import _ from "lodash";
```

Comment:
I vote we just use `babel-plugin-lodash` and do lodash imports the easy way.
(import like above, then babel takes care of the efficient module imports)
https://github.com/lodash/babel-plugin-lodash

`meteor npm i --save-dev babel-plugin-lodash`

```js
// .babelrc
{
  "plugins": ["lodash"]
}
```